examples/react-native/v12/TestApp/__tests__/App.test.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+import {render} from '@testing-library/react-native';
+
+// Note: import explicitly to use the types shiped with jest.
+import {test, expect} from '@jest/globals';
```

Comment:
```suggestion
// Note: import explicitly to use the types shipped with jest.
import {test, expect} from '@jest/globals';
```

Is there a reason we're going with explicit imports in each test file rather than using global variables? `@types/jest` provides globals out of the box but I think might have some lag behind the real jest types. Maybe we can import these and define some global vars in the jest setup file?

Reply:
Oh, yeah. Good point! As mentioned above, this is the test file that came with the React Native template app and I hadn't cleaned it up yet.

I'll set up some global vars for testing types. Thanks for pointing this out!